src/django_idom/components.py

```diff
@@ -68,56 +68,83 @@ async def async_render():
                         "src": reverse("idom:view_to_component", args=[dotted_path]),
                         "loading": "lazy",
                     }
-                )
+                )  # type: ignore
             )
             return
 
         # Render Check 2: Async function view
         elif iscoroutinefunction(view):
-            view_html = await view(request_obj, *args, **kwargs)
+            view_html = await view(request_obj, *_args, **_kwargs)  # type: ignore
```

Comment:
This seems to be a problem because the type of `view` ought to be `type[View]` instead of a `View` instance.